OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -321,12 +323,39 @@ protected virtual void Tick(Actor self)
 
 			var oldCachedPosition = cachedPosition;
 			cachedPosition = self.CenterPosition;
-			isMoving = (oldCachedPosition - cachedPosition).HorizontalLengthSquared != 0;
-			isMovingVertically = (oldCachedPosition - cachedPosition).VerticalLengthSquared != 0;
+			UpdateIsMoving(self, (oldCachedPosition - cachedPosition).HorizontalLengthSquared != 0, (oldCachedPosition - cachedPosition).VerticalLengthSquared != 0);
```

Comment:
Style nit: this line is a bit too long IMO, so would be better to pull out `var horizontal = ...; var vertical = ...; UpdateIsMoving(self, horizontal, vertical);`.

Reply:
This probably no longer applies with my comment below.